binderhub/tests/test_build.py

```diff
@@ -21,7 +21,7 @@
 
 # We have optimized this slow test, for more information, see the README of
 # https://github.com/binderhub-ci-repos/minimal-dockerfile.
-@pytest.mark.asyncio(timeout=900)
+@pytest.mark.asyncio
```

Comment:
According to https://github.com/pytest-dev/pytest-asyncio/issues/215 the timeout argument was only a proposal.
This means the argument would've been silently ignored until additional error checking was added in https://github.com/pytest-dev/pytest-asyncio/pull/886